File: app/src/main/java/info/nightscout/androidaps/plugins/PumpInsightLocal/app_layer/configuration/OpenConfigurationWriteSessionMessage.java
package info.nightscout.androidaps.plugins.PumpInsightLocal.app_layer.configuration;

import info.nightscout.androidaps.plugins.PumpInsightLocal.app_layer.AppLayerMessage;
import info.nightscout.androidaps.plugins.PumpInsightLocal.descriptors.MessagePriority;
import info.nightscout.androidaps.plugins.PumpInsightLocal.app_layer.Service;

public class OpenConfigurationWriteSessionMessage extends AppLayerMessage {

    public OpenConfigurationWriteSessionMessage() {
        super(MessagePriority.NORMAL, false, false, Service.CONFIGURATION);
    }
}

File: pump/insight/src/main/kotlin/app/aaps/pump/insight/app_layer/configuration/OpenConfigurationWriteSessionMessage.kt
package app.aaps.pump.insight.app_layer.configuration

import app.aaps.pump.insight.app_layer.AppLayerMessage
import app.aaps.pump.insight.app_layer.Service
import app.aaps.pump.insight.descriptors.MessagePriority

class OpenConfigurationWriteSessionMessage : AppLayerMessage(MessagePriority.NORMAL, false, false, Service.CONFIGURATION)